e2e/steps_file.js

```diff
@@ -15,6 +15,7 @@ const normalizeCaseId = caseId => caseId.toString().replace(/\D/g, '');
 const baseUrl = process.env.URL || 'http://localhost:3333';
 const signedInSelector = 'exui-header';
 const signedOutSelector = '#global-header';
+let currentUser = {};
```

Comment:
hmm, how does it works when multiple executors?

Reply:
seems to work fine, was worried about it as well but it never seemed to log in unnecessarily. Will run again though to check.

Reply:
it is a bit slippery, assumes that all login/logout happens via function signIn from this file

Reply:
https://codecept.io/parallel/

should be fine as long as we use run-multiple